test-apps/display-performance-test-app/src/backend/WebMain.ts

```diff
@@ -58,6 +58,10 @@ function startWebServer() {
       browser = arg;
     else if (arg === "headless")
       chromeFlags.push("--headless");
+    else if (arg === "egl"){
```

Comment:
@pmconne Is there a way to get chrome to pick up the GPU without these changes ?

Reply:
Looks good to me.